styles.css

```diff
@@ -0,0 +1,857 @@
+    margin: 0 10% 0 10%;
+  }
+
+  .program-activity {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    height: 40vh;
+  }
+
+  .program-icon {
+    padding-top: 7%;
+    width: 60%;
+  }
+
+  .desktop-see {
+    border-bottom: 2px solid #fff;
+    margin: 1%;
+    padding-bottom: 0;
+    margin-bottom: 4.5%;
+  }
+
+  .daily-program {
+    font-size: 1.8rem;
+    margin-bottom: 0;
+    padding: 1.7rem;
+  }
+
+  .headliners > .daily-div > .daily-program {
+    margin: 0;
+  }
+
+  .partners-img {
+    width: 80vw;
+    margin: 0 10% 0 10%;
+    justify-content: space-around;
+  }
+
+  .partners-img > img {
+    padding: 0;
+    height: 15vh;
+    width: auto;
+  }
+
+  #smaller {
+    height: 6vh;
+    padding-top: 4%;
+  }
+
+  .partners-div > h3 {
+    font-size: 1.8rem;
+    margin-bottom: 0;
+    padding: 1.7rem;
+    color: #a5a7ac;
+  }
+
+  .footer {
+    height: 20vh;
+    display: flex;
+    justify-content: space-between;
+    width: 100vw;
+    padding: 1% 10% 3% 10%;
+  }
+
+  .footer > img {
+    height: 8vh;
+    padding-right: 15%;
+  }
+
+  .footer > p {
+    font-size: 0.6rem;
+    padding-top: 3%;
+  }
+
+  .about-container {
+    padding-top: 5%;
+  }
+
+  .no-display {
+    display: none;
+  }
+
+  .about-container > h3 {
+    padding-bottom: 5%;
+  }
+
+  .about-container-p {
+    font-size: 1.2rem;
+    width: 70vw;
+    line-height: 2rem;
+  }
+
+  .contact-us > p {
+    font-size: 1.2rem;
+    width: 100%;
+    line-height: 2.2rem;
+  }
+
+  .contact-us > p > span {
+    text-decoration: underline;
+    padding: 1%;
+  }
+
+  .logo-description > .desktop-see {
+    margin: 2%;
+    font-size: 1.2rem;
+    line-height: 2.2rem;
+    width: 60vw;
+  }
+
+  .logo-description > img {
+    height: 17vh;
+    width: auto;
+    padding: 3%;
+    border: 1px solid #a5a7ac;
+    background-color: #a5a7ac1b;
+  }
+
+  .logo-description > h3 {
+    font-size: 1.8rem;
+    margin-bottom: 0;
+    padding: 1.7rem;
+  }
+
+  .logo-description > p {
+    padding: 0;
+  }
+
+  .about-page > .front-page > .wrapper > .partners-div {
+    display: none;
+  }
+
+  .see-past > h3 {
+    font-size: 1.8rem;
+    margin-bottom: 0;
+    padding: 1.7rem;
+  }
+
+  .see-past > p {
+    padding: 0;
+    font-size: 1.2rem;
+    width: 100%;
+    line-height: 2.2rem;
+  }
+
+  .see-past > div > .img-1,
+  .see-past > div > .img-2 {
+    height: 20vh;
+    width: 35vw;
+    margin: 1%;
+  }
+
+  .past-images {
+    display: flex;
+    margin-bottom: 5%;
+  }
+
+  .about-footer {
+    background-color: #2b2e33;
+    color: #fff;
+    padding: 3% 10% 3% 10%;
+  }
+
+  .about-footer > img {
+    margin-right: 10%;
+    margin-left: 10%;
+    height: 10vh;
+  }
+
+  #dynamicHeadliners {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    margin: 3%;
+  }
+
+  .dynamic-bands {
+    width: 35vw;
+  }
```

Comment:
- [ ] As stated in the project requirement You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors,Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design) and Layout: composition and space between elements. **So Kindly adjust the background transparent image for the each band members just like the Design Guidelines :**-
- The Design Guidelines for the Featured speakers section Look likes this : 
 
![feature](https://github.com/agneta1991/Capstone_project/assets/93598189/09bdded4-c40a-41ec-8b93-15459e6578d5)

- Your Project band Headliners section Look likes this : 

![band](https://github.com/agneta1991/Capstone_project/assets/93598189/0cf58359-4534-4aef-a31e-15e22489820a)